### ISSUE TITLE
Add --null option for command line tool

### ISSUE THREAD

**Issue by achalddave** [tqdm/tqdm#161]
It would be nice to be able to use tqdm like so: `find ... -print0 | tqdm --null | xargs -0 ...`.

A more general `--delimiter` would also be nice, but if i understand correctly, a `--null` option would still be necessary as it's not possible to pass a null character to `--delimiter`.

**Comment by lrq3000** [comment]
Could you please detail what you mean for me (a poor Windows user :-( ), more specifically what would be the input and the expected behavior of `tqdm`?

**Comment by casperdcl** [comment]
I get it - will implement shortly. @lrq3000 UNIX utils tend to have an option to delimit with null chars rather than newlines.

**Comment by casperdcl** [comment]
@achalddave in the meanwhile you should also be able to do

``` sh
find -name "*.py" -print0 | tr '\0' '\n' | tqdm >>/dev/null
```

**Comment by achalddave** [comment]
Thanks! I used a similar workaround for now, but I figured I'd put in the feature request :)

**Comment by lrq3000** [comment]
@achalddave And you did well ;) Casper is doing an amazing job with the CLI interface!

**Comment by casperdcl** [comment]
merged in #162 
# example usage
``` sh
find ... -print0 | tqdm -delim '\0' | ...
seq -s, 10 | tqdm -delim , | ...
cat *.tsv | tqdm -delim '\t' | ...
```

**Comment by lrq3000** [comment]
:+1:
